libvast/src/system/spawn_source.cpp

```diff
@@ -35,10 +35,6 @@ spawn_source(node_actor::stateful_pointer<node_state> self,
     = make_transforms(transforms_location::server_import, args.inv.options);
   if (!transforms)
     return transforms.error();
-  // TODO: Remove this check once ch25395 is implemented.
```

Comment:
Please rename the original changelog entry for transforms to include this PR number in the filename.